### PR TITLE
POL-290 Azure Policy Audit

### PR DESCRIPTION
**Pull request by srpomeroy**
### Description

The initial release of the Azure Policy Audit policy template.

### Issues Resolved

n/a

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: compliance/azure/azure_policy_audit/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- The policy leverages the Azure API to check for assigned policies in all subscriptions the service principal has access to.
+- Assigned policies are compared against the list of policies provided.
+- An email is sent with details on which policies are assigned and not assigned to each subscription.
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Policy Names* - A list of Azure Policies. **Case sensitive.**
```
Comment (cdwilhelm) [inline compliance/azure/azure_policy_audit/README.md:17]: Does it need to be case sensitive?  I suggest converting to lowercase both values when evaluating.

File: compliance/azure/azure_policy_audit/azure_policy_audit.pt [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+define sys_log($subject, $detail) do
+    rs_cm.audit_entries.create(
+      notify: "None",
+      audit_entry: {
+        auditee_href: @@account,
+        summary: "Azure Disallowed Regions "+ $subject,
+        detail: $detail
+      }
+    )
+end
```
Comment (cdwilhelm) [inline compliance/azure/azure_policy_audit/azure_policy_audit.pt:270]: this can be removed

Comment (srpomeroy) [reply compliance/azure/azure_policy_audit/azure_policy_audit.pt]: Removed.

File: compliance/azure/azure_policy_audit/README.md (hunk 1 above)
Comment (srpomeroy) [reply compliance/azure/azure_policy_audit/README.md]: Updated find logic to compare lower case values.

**Pull request merged**